omnibus/config/software/python3.rb

```diff
@@ -65,12 +65,12 @@
     dependency "vc_ucrt_redist"
 
     source :url => "http://s3.amazonaws.com/dd-agent-omnibus/python-windows-#{version}-x86.zip",
-            :sha256 => "c9ccf9cd81c06e49cb3186bef1e769a6b9da58d00dbb780f0185fbb2e8efba91"
+            :sha256 => "5250eb94858da7e0c07163cbe0400253171a2498502ec9ad4078b04802c54040"
   else
 
     # note that startring with 3.7.3 on Windows, the zip should be created without the built-in pip
     source :url => "https://s3.amazonaws.com/dd-agent-omnibus/python-windows-#{version}-amd64.zip",
-         :sha256 => "ce1782db64be81aa81e8a38102b4850ee03a0b30bf152a7d2b4b36a7a6e0c381"
+         :sha256 => "58ac525fecdd96735a779bd094096ee42bc4bf5b6f0bbad0035db700f7bcfd98"
```

Comment:
If we want to ship `3.8.1`, we need to prepare, upload and use a new Windows Python zip for 3.8.1 here (both for amd64 and x86).

Reply:
Yup, there is this and the python used for the unit tests, I was about to ping platform / @derekwbrown :D